campaignion_tracking/package/src/ga4.js

```diff
@@ -73,6 +73,13 @@ export class GA4Tracker {
     this.donationSubscription = this.tracker.subscribe('donation', this._dispatch)
   }
 
+  /**
+   * Copy gtag function as required by GA4.
```

Comment:
How about

> Generate an arguments object and pass it to the dataLayer.

This would tell us why the function is needed as well.

(github doesn’t provide me with the option to suggest an edit for this line.)

Reply:
Have updated the comment. To me the most important info is still that it's a copy of the gtag function.